common/package_manager.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+Module for working with packages
+
+"""
+
+import platform
+import subprocess
+
+from enum import Enum
+
+
+class OSType(Enum):
+    """
+    Container for os version
+    """
+
+    CENTOS = "centos"
+    DEBIAN = "deb"
+
+
+CMD_PATTERN = {
+    "INSTALL": {
+        "deb": "dpkg -y install {pkg}",
+        "centos": "yum -y install {pkg}"
+    },
+    "UNINSTALL": {
+        "deb": "aptitude -y remove {pkg}",
+        "centos": "yum -y remove {pkg}"
+    },
+    "CHECK_INSTALLED": {
+        "deb": "dpkg --list | grep {pkg}",
+        "centos": "yum list installed | grep {pkg}"
+    }
+}
+
+
+def install_pkg(pkg_path):
+    """
+
+    :param pkg_path: path to pkg to install
+    :type: pathlib.Path
+
+    :return: Flag whether pkg installed
+    :rtype: bool
+    """
+    returncode = execute_command(get_install_cmd(pkg_path), sudo=True)
+    if not returncode:
+        return True
+    return False
+
+
+def uninstall_pkg(pkg_name):
+    """
+
+    :param pkg_name: name of pkg to uninstall
+    :type: String
+
+    :return: Flag whether pkg uninstalled
+    :rtype: bool
+    """
+    returncode = execute_command(get_uninstall_cmd(pkg_name), sudo=True)
+    if not returncode:
+        return True
+    return False
+
+
+def is_pkg_installed(pkg_name):
+    """
+    Check whether pkg is installed
+
+    :param pkg_name: pkg name
+    :type: String
+
+    :return: Flag whether pkg is installed
+    :rtype: bool
+
+    """
+    cmd = CMD_PATTERN["CHECK_INSTALLED"].get(get_os_version()).format(pkg=pkg_name)
+    returncode = execute_command(cmd)
+    if not returncode:
+        return True
+    return False
+
+
+def get_install_cmd(pkg_path):
```

Comment:
get_install_cmd functions are not needed :)
or it can be one private function with two parameters